### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by 0hsn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,49 @@
...
+- [http: POST with header and bearer token](https://chkware.github.io/Examples/http-examples#request-with-bearer-authentication-header)
+- [http: POST form submit](https://chkware.github.io/Examples/http-examples#request-with-form)
+- [http: POST with JSON body](https://chkware.github.io/Examples/http-examples#request-with-json-body)
+- [http: PUT with file upload](https://chkware.github.io/Examples/http-examples#request-with-file-upload)
+
+---
+
+### Ask anything
+
+Please use tag `chkware-cli` with the question on [stackoverflow.com](https://stackoverflow.com/questions/tagged/chkware-cli) to ask or discuss anything. Follow the tag if you want to help others with your knowledge.
```
Comment (arifszn) [inline README.md:51]: Maybe issue tracker link also?
```markdown
### How do I report feedback and issues?

You can [file an issue](https://github.com/chkware/vscode-ext/issues) on the GitHub repo.
```

File: README.md [hunk 2]
```diff
@@ -1,17 +1,49 @@
...
+## Quick start
 
-- Run `npm install` to install dependencies for the first time
-- Run `npm run watch`
-- Press `f5`
-- Wait for the new window to be opened
-- Open command palette by pressing `ctrl` + `shift` + `p`
-- Search for `chkware` commands
-- After code changes, run `Developer: Reload Window` in the new window from command palette
+Get started writing `chkware` configurations with VS Code in three steps:
```
Comment (arifszn) [inline README.md]:
```suggestion
Get started writing `chkware` configurations with VS Code in two steps:
```

**Review by arifszn — CHANGES_REQUESTED** [review]

**Comment by arifszn** [comment]
Untitled 2

Can this gif be used?

**Comment by 0hsn** [comment]
> Untitled 2
> 
> Can this gif be used?

Yes, good. but, can you check why syntax coloring is not working? It was supposed to set `YAML` syntax, therefore coloring should be there. or i am getting it wrong.

**Comment by arifszn** [comment]
> Yes, good. but, can you check why syntax coloring is not working? It was supposed to set `YAML` syntax, therefore coloring should be there. or i am getting it wrong.


As this was already an opened markdown file, the syntax didn't work. But if no opened file were there, it would create a new YAML file. Changing an already opened file's formatting language programmatically isn't possible right now.

Untitled 5

**Comment by 0hsn** [comment]
> As this was already an opened markdown file, the syntax didn't work. But if no opened file were there, it would create a new YAML file. Changing an already opened file's formatting language programmatically isn't possible right now.

It's okay if the feature is not available. But, at least the gif looks okay now

**Review by arifszn — APPROVED** [review]

**Pull request merged**